VogtPayroll4/EmployeePayrollConsoleReader.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace VogtPayroll4
+{
+    class EmployeePayrollConsoleReader
```

Comment:
Oops I said ConsoleReader, but this is a FileReader. EmployeePayrollFileReader? Something like that would be better.